docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,783 @@
+
+In this step, you will configure the Universal Forwarder to receive audit logs
+from the Teleport Event Handler plugin and forward them to Splunk. The Event
+Handler sends audit logs as HTTP POST requests with the content type
+`application/json`.
+
+We will assume that you assigned `$SPLUNK_HOME` to `/opt/splunkforwarder` when
+installing the Universal Forwarder. 
+
+<Details title="Finding your $SPLUNK_HOME">
+
+To find your `$SPLUNK_HOME`, run the following command to see the location your
+Universal Forwarder service definition, which the init system systemd uses to
+run the Universal Forwarder:
+
+```code
+$ sudo systemctl status SplunkForwarder.service
+● SplunkForwarder.service - Systemd service file for Splunk, generated by 'splunk enable boot-start'
+     Loaded: loaded (/lib/systemd/system/SplunkForwarder.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2022-10-07 15:57:37 UTC; 2h 18min ago
+   Main PID: 1772 (splunkd)
+      Tasks: 53 (limit: 2309)
+     Memory: 70.8M (limit: 1.8G)
+     CGroup: /system.slice/SplunkForwarder.service
+             ├─1772 splunkd --under-systemd --systemd-delegate=yes -p 8089 _internal_launch_under_systemd
+             └─1810 [splunkd pid=1772] splunkd --under-systemd --systemd-delegate=yes -p 8089 _internal_launch_under_systemd [process-runner]
+```
+
+View the file at the path shown in the `Loaded:` field. Your `$SPLUNK_HOME`
+will include the filepath segments in `ExecStart` before `bin`.
+
+</Details>
+
+### Create an index for your audit logs
+
+Create an index for your Teleport audit logs by visiting the home page of the
+Splunk UI and navigating to **Settings** > **Indexes**. Click **New Index**.
+Name your index `teleport-audit-logs` and assign the **Index Data Type** field
+to "Events". 
+
+![Creating an Index](../../../../docs/img/enterprise/plugins/splunk/new-index.png)
+
+The values of the remaining fields, **Max raw data size** and **Searchable
+retention (days)** depend on your organization's resources and practices for log
+management.
+
+Click "Save".
+
+### Create a token for the Universal Forwarder
+
+The Universal Forwarder authenticates client traffic using a token. To generate
+a token, visit the home page of the Splunk UI. Navigate to **Settings** > **Data
+inputs** In the **Local inputs** table, find the **HTTP Event Collector** row and
+click **Add new** 
+
+Enter a name you can use to recognize the token later so you can
+manage it, e.g., `Teleport Audit Events`. Click **Next**.
+
+Next to the **Source type** field, click **Select**. In the **Select Source
+Type** dropdown menu, click **Structured**, then **_json**. Splunk will index
+incoming logs as JSON, which is the format the Event Handler uses to send logs
+to the Universal Forwarder.
+
+![Create a Token](../../../../docs/img/enterprise/plugins/splunk/new-token.png)
+
+Beside **Index** select the `teleport-audit-logs` index you created earlier.
```

Comment:
Does "Beside **Index**" refer to the "Select Allowed Indexes" text in the screenshot above?

Reply:
I've made this clearer